src/datajunction/console.py

```diff
@@ -24,6 +32,7 @@
 
 from datajunction import __version__
 from datajunction.cli import compile as compile_
+from datajunction.cli import add_database as add_database_
```

Comment:
No need for the underscore here, the only reason we used in line 34 is because `compile` is a Python builtin. Conventionally we use the underscore suffix on variables that would shadow builtins or reserved keywords.

```suggestion
from datajunction.cli import add_database